lib/plug.ex

```diff
@@ -15,16 +15,27 @@ if Code.ensure_loaded?(Plug) do
       client_version_header = Keyword.get(opts, :version_header, "x-api-version")
       metadata_formatter = Keyword.get(opts, :metadata_formatter, MetadataFormatter)
       exclude_routes = Keyword.get(opts, :exclude_routes, [])
-      {level, metadata_formatter, exclude_routes, client_version_header}
+      include_variables = Keyword.get(opts, :include_variables, ["id"])
```

Comment:
```suggestion
      (vars|params|...)_whitelist = Keyword.get(opts, :(vars|params|...)_whitelist, ["id"])
```

Reply:
They are not abbreviating usually, there are no well-known name for this unless "variables"

Reply:
We can name it as `include_graphql_variables` to eliminate ambiguity, what about that?